content/user/advanced-modules/PDFTemplates.adoc

```diff
@@ -0,0 +1,116 @@
+Page Size/ Orientation:: If necessary, select the size and orientation of the template page. Then default is A4 size and portrait page orientation.
+Inserting into a Field::  This option allows you to add the necessary variable to the template, which later (in the process of creating a document based template) will be replaced with the necessary data from the SuiteCRM+
+To insert a variable: :::
+*	In the left combo box, select the module from which you want to get information.
+*	In the middle combo box, select the name of the field of the selected module, then the name of the necessary variable corresponding to the selected field so it appears in the right field.
+*	In the test of the template, specify the place where the selected variable should be added and click the button {btn}[Insert]
+
+The following picture shows an example of adding the variable * $aos_invoices_number*, which, when creating the document based on the current template, will be automatically replaced with the account number.
+
+image:PDFinvoicenumber.png[Insert field in template]
+
+If it is necessary to add data in the template, for example, listing positions in the Offer or Invoice, then such data should be arranged in a table:
+
+image:QPD.png[Insert Table into Template]
+
+
+== Creating Headers and Footers
+
+At the bottom of the Template Editing Form are two fields that allow you to customize the header and footer for the current template:
+
+image:Pagenoanddate.png[Creating headers and footers]
+
+Footers are able to use special variables:
+
+*	*{PAGENO}*,  Showing the current page number.
+*	*{DATE D.m.Y}*,  Showing the current date in the required format
+
+== Setting Page Parameters
+
+Under the header and footer there are several fields that allow you to customise different page parameters, such as the size of the indents from the page edges of the header and footer fields:
+
+image:margins.png[Setting page parameters]
+
+== Creating PDF Documents Based on a Template
+
+ .	When creating a PDF document for modules *Counterparties*, *Contacts* and *Preliminary Contacts* use the menu item *Print as PDF* from the action menu of the View module of the corresponding module:
+
+image:PDF-Contact.png[Creating PDF documents based on a template]
+
+If several templates were created for the current module, they will be presented in the form of a list, from which you need to select the desired position. :
+
+image:PDF selection.png [Select a template from the list]
+
+If only one template was created for the current module, then the necessary document will be immediately created on its basis.
+
+If you need to create several similar documents for several module entries at once, select the necessary entries in the List Form and then use the * Create Letter (PDF) * from the action menu:
```

Comment:
If those asterisks are for `bold` they can't have spaces. So it should be `*Create Letter (PDF)*`